### PR TITLE
Fix/add conditionals

### PR DESCRIPTION
**Pull request by ADavidson02**
## Is this a fix or a feature?
Feature

## What is the change?
Conditionals have been added for the input fields for updating data.

## Where should the reviewer start?
src/scripts.js line 54
src/css/style.scss line 196
src/index.html  line 49

## How should this be tested?
As a user try and press any of the update data buttons.
An error alert should appear. Once the data is entered the button will then be clickable. 

@Shawntru 
@adavidson02
@sarah01123

### CODE REVIEW

File: src/css/style.scss [hunk 1]
```diff
@@ -192,3 +192,17 @@ h1 {
   font-weight: bold;
   font-family: $document-font-family;
 }
+
+.new-hydration-button, .new-activity-button,
```
Comment (saraho1123) [inline src/css/style.scss:196]: Thank you for styling these!

**Review by Shawntru — APPROVED** [review]
Great

File: src/scripts.js [hunk 2]
```diff
@@ -149,10 +158,29 @@ function updatePageHydration (userNowId, todaysDate) {
 
 function submitNewActivity(userNowId) {
   let todaysDate = getTodaysDate()
-  requests.postActivityData(userNowId, todaysDate, +newStepsInput.value, +newActiveMinutesInput.value, +newStairsInput.value)
+  let newStepsActivityData = newStepsInput.value;
+  let newMinutesActivityData = newStairsInput.value;
+  let newStairsActivityData = newActiveMinutesInput.value;
+  if (!newStepsActivityData) {
+    return alert('Please enter a number of steps');
+  } else if (newStepsActivityData > 20000) {
```
Comment (saraho1123) [inline src/scripts.js:166]: I'm making it my goal to go over 20,000. ;)

**Review by saraho1123 — APPROVED** [review]
Awesome! Thanks for doing this. And I'm sorry for the trouble with github!

**Pull request merged**